styles/mixins/_typography.scss

```diff
@@ -5,6 +5,11 @@
     line-height: $line-height-xl;
     font-weight: $font-weight-bold;
 }
+@mixin heading-1-1 {
```

Comment:
❌ 
Let's name it `heading-1-xxl` so that it's more clear what the mixin does.

Also, please don't forget to separate blocks with empty lines.

Reply:
done